nativelib/src/main/scala/scala/scalanative/regex/Matcher.scala

```diff
@@ -307,17 +307,46 @@ final class Matcher private (private var _pattern: Pattern) {
     }
   }
 
+  def regionSpansInput(): Boolean = {
+    val _regionLength = _regionEnd - _regionStart
+    _regionLength == _inputLength // if true, implies regionStart is 0
```

Comment:
```suggestion
    // if true, implies regionStart is 0
    _inputLength == _regionEnd - _regionStart
```
Maybe with this change comment should change?

Reply:
Re-worked as suggested and changed comment.  At least the former is an improvement. 
I do not know if I captured my intent in the changed comment.  I know the intent now
but would for me and those following me to have a clue six months on.

Reply:
Later:  I re-worked it again and think I got the code to convey the intent, rather than
obscure code requiring a comment.  I also got rid of what some people, not I, would 
consider extraneous curly braces.